langchain/vectorstores/chroma.ts

```diff
@@ -0,0 +1,194 @@
+import fs from "fs/promises";
+import path from "path";
+import type {
+  ChromaClient as ChromaClientT,
+} from "chromadb";
+import { ChromaClient as ChromaClientO } from "chromadb";
```

Comment:
I think this will raise an exception if this file is imported and you don't have chromadb installed, we can probably just remove this line and rely on line 17?